content/rs/installing-upgrading/configuring/change-location-socket-files.md

```diff
@@ -6,66 +6,62 @@ alwaysopen: false
-You can change the location of the socket files anytime, but to limit
-downtime, it is best to perform this change when you are
-[installing]({{< relref "/rs/installing-upgrading/downloading-installing.md" >}})
-or
-[upgrading]({{< relref "/rs/installing-upgrading/upgrading.md" >}})
-Redis Enterprise Software to version 4.5.0.-51 or higher. If this change
-needs to be performed immediately on an existing cluster, see the
-instructions below.
+There are two default locations for the socket files:
 
-## Changing the Location on an Existing Redis Enterprise Software (RS) Installation
+- `/tmp` - In clean installations of RS version lower than 5.2.2
+- `/var/opt/redislabs/run` - In clean installations of RS version 5.2.2 and higher
 
-You must be running RS version 4.5.0-51 or higher, and with the sock
-files location is still under the default location of /tmp. (This is
-where they go when install.sh is executed without -s flag.)
+When you upgrade from a RS version lower than 5.2.2 to 5.2.2 and higher, the socket files
+are not moved to the new location by default. During [installation]({{< relref "/rs/installing-upgrading/downloading-installing.md" >}})
+you can specify a custom location for the socket files, but after installation
+you must use this procedure to move the socket files.
 
-1. Execute this command **on all nodes**:
+To change the location of the socket files:
+
+1. On each node in the cluster, run:
 
     ```src
-    $ sudo rlutil create_socket_path socket_path=/var/run/redislabs
+    $ sudo rlutil create_socket_path socket_path=/var/opt/redislabs/run
     ```
 
-1. Execute this command **on the master node**:
+1. Identify the master node:
+    1. On any node in the cluster, run: `rladmin status nodes`
+    1. Find the node that has the **master** role.
+1. On the master node, run:
 
     ```src
-    $ sudo rlutil set_socket_path socket_path=/var/run/redislabs
+    $ sudo rlutil set_socket_path socket_path=/var/opt/redislabs/run
     ```
 
-1. Execute this command **on all nodes in a serial manner**:
+1. On each node in the cluster one at a time, run:
 
     ```src
     $ sudo service rlec_supervisor restart
     ```
 
-1. There are two ways to do the next step depending on your uptime
-    requirements.
-    1. Restart each database in the cluster, but you will incur
-        downtime. This is by far the easiest option to execute.
-
+1. To restart all of the shards in the cluster, either:
+    - Restart each database in the cluster:
+    
+        {{% warning %}}
+        Database restart can cause interruptions in data traffic.
+        {{% /warning %}}
         ```src
         $ rladmin restart db <db name>
         ```
 
-    1. Execute the following three commands **for each** master/slave
-        shards during a maintenance window. All three of these commands
-        **must** be executed **on the node hosting the shard being
-        operated on**:
+    - For each master/slave shard for each database in the cluster, run:
```

Comment:
@RoeyPrat  - what's the difference between this way and the previous one? do you know why we have both? It seems to me that the first one should be enough

Reply:
@bmansheim - also @RoeyPrat doesn't see any reason for having the second option- option #1 should be good enough. However, it worth checking who performed the original documentation commit and get more input.
BTW- @RoeyPrat  suggested to write the shards restart procedure/s in a separate page and add reference from all the relevant pages (like this one). I think it can be useful in case we have more than one place that explains this procedure. Otherwise, I'm not sure it worth the efforts (for now)

Reply:
@Paz-Yanover This content was brought from WordPress so I don't know who added it. Do you want to send it to someone else to review, like QA?

Reply:
I don't think QA is relevant here. Maybe @ushachar  has some insights here.

Reply:
@ushachar Should we include the "Restart each master and slave shard" instructions or just give the "Restart each database" instructions?

Reply:
I think we should stay with 'each master & slave' - otherwise folks might try and use the rladmin restart....

Reply:
@Paz-Yanover I clarified with Uri. He said Roey is the authoritative answer. I'll merge the PR.

Reply:
@ushachar - I didn't understand your comment. The original documentation talks about two options to "restart all of the shards in the cluster": (1) by using "rladmin restart db" (2) by using redis_ctl stop; rladmin failover shard; redis_ctl stop
Do you think that we need also option #2 and the first option is not enough / good?

Reply:
As I told @bmansheim today - @RoeyPrat has the authoritative answer on this as he implemented the feature and should know what's needed...

Reply:
@RoeyPrat @bmansheim - so, can we remove the second suggestion to restart all the shards in the cluster?

Reply:
@Paz-Yanover That's my understanding. I removed the instructions for restarting all shards.

Reply:
yes. 😆